### ISSUE TITLE
Can't verify CSRF token authenticity

### ISSUE THREAD

**Issue by BastianGrahm** [disraptor/disraptor#10]
When clearing browser cache or simply trying to log in from a incognito browser while being logged in normally "Unknown Error" appears in the Log In form.
The rails production log states the following:
```
Started POST "/session" for 141.54.132.159 at 2020-11-13 10:17:39 +0000
Processing by SessionController#create as */*
  Parameters: {"login"=>"<USERNAME>", "password"=>"[FILTERED]", "second_factor_method"=>"1", "timezone"=>"Europe/Berlin"}
Can't verify CSRF token authenticity.
  Rendering text template
  Rendered text template (Duration: 0.0ms | Allocations: 1)
Filter chain halted as :verify_authenticity_token rendered or redirected
Completed 403 Forbidden in 3ms (Views: 1.4ms | ActiveRecord: 0.0ms | Allocations: 1046)
```

Further information:
[[1] Stackoverflow](https://stackoverflow.com/questions/35181340/rails-cant-verify-csrf-token-authenticity-when-making-a-post-request) 
[[2] Rails](https://guides.rubyonrails.org/security.html#cross-site-request-forgery-csrf)

Rails method :verify_authenticity_token is called and returns a 403 Forbidden code.

__This is no issue when Disraptor is not rerouting the `/` route!__

**Comment by BastianGrahm** [comment]
This may also prevent a user from logging out.

**Comment by mam10eks** [comment]
Notes:

- The problem is caused by jquery that is embedded into tira web-pages.

- Possible solution:
  - Remove loading of jquery
     - Some functionality might be broken
     - Starting/Stopping might broken
    - Is this really broken?
      - Because discourse itself also loads jquery
- Update version of jquery

What we need to do:
- Solve this issue
- Document such mistakes

**Comment by BastianGrahm** [comment]
jQuery removal in Tira resolves the issue. 
Replacing the old jQuery version used by Tira by the latest version (jQuery 3.5.1) does break the login button completely. This points more towards the removal-idea.

**Comment by BastianGrahm** [comment]
Problem looks to be fixed now in every occasion it appeared earlier. With this fix also problem #11 `Workflow Login -> Logout -> Login is broken` does not appear anymore. No further work should be needed here.
